src/docs/resources/faq.md

```diff
@@ -371,6 +371,16 @@ binaries within the IPA, making the compression less efficient
 (see the [iOS App Store Specific Considerations][]
 section of Apple’s [QA1795][]).
 
+Since Flutter has started bundling bitcode, the size of the release
+Flutter.framework has vastly increased. This is because the release binary now
```

Comment:
Talk about what this means for people who don't see .frameworks and see .app or .ipas via xcode or flutter build ios. i.e. is this an add-to-app specific note?